README.md

```diff
@@ -73,6 +73,7 @@ with the following options:
 - List of extensions: `*.twig`
 - Invocation:
     - Parser command: `<project>/vendor/bin/twig-gettext-extractor --sort-output --force-po -o %o %C %K -L PHP --files %F`
+        - (Windows user having problems? Try to use this instead `C:\\Users\path\to\your\project\vendor\bin\twig-gettext-extractor.bat --sort-output --force-po -o %o %C %K -L PHP --files %F`)
```

Comment:
So you renamed [twig-gettext-extractor](https://github.com/umpirsky/Twig-Gettext-Extractor/blob/master/twig-gettext-extractor) to `twig-gettext-extractor.bat`? This is odd, because it's PHP file.

I find your path confusing also because it refers to non existing bat file.

No, it does not replace it, but we can write explanation below:

Replace `<project>` with absolute path to your project.

I appreciate your efforts to make things clear for windows users, I'm just afraid not to confuse them with referring to non existing path/file.